packages/uploadthing/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+import type { PackageJson } from "type-fest";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageJson = require("../package.json") as PackageJson;
+if (!packageJson.version) throw new Error("no version found in package.json");
+export const UPLOADTHING_VERSION = packageJson.version;
```

Comment:
Thank you for this, I have been saying for a bit that we should really have a centralized constants file, but had not gotten around to actually doing it.